packages/datadog-plugin-http/test/server.spec.js

```diff
@@ -131,5 +132,38 @@ describe('Plugin', () => {
         expect(() => res.emit('finish')).to.not.throw()
       })
     })
+
+    describe('with a blocklist configuration', () => {
+      beforeEach(() => {
+        return agent.load('http', { blocklist: '/health' })
+          .then(() => {
+            http = require('http')
+          })
+      })
+
+      beforeEach(done => {
+        const server = new http.Server(listener)
+        appListener = server
+          .listen(port, 'localhost', () => done())
+      })
+
+      it('should drop traces for blocklist route', done => {
+        const spy = sinon.spy(() => {})
+
+        agent
+          .use((traces) => {
+            spy()
+          })
+          .then(done)
```

Comment:
If a trace is received before 100ms, this test will pass. As an alternative here, try removing the `then`. The only success case here should be when the spy has not been called after 100ms.